src/fedora/39/amd64/Dockerfile

```diff
@@ -66,4 +66,5 @@ RUN dnf --setopt=install_weak_deps=False install -y \
 # Dependencies for Aspnetcore
 RUN cd ~ && \ 
     curl -sL https://rpm.nodesource.com/setup_20.x
-RUN dnf install -y nodejs && dnf clean all
\ No newline at end of file
+RUN dnf install -y nodejs && dnf clean all
+ENV NO_UPDATE_NOTIFIER=true
```

Comment:
This should be applied consistently across all the Node-related Dockerfiles.

Reply:
Done